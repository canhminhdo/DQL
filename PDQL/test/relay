#!/bin/sh

DIR=../test
cd ../examples && maude < relay.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/relay.out
diff $DIR/relay.expected $DIR/relay.out
rm $DIR/relay.out