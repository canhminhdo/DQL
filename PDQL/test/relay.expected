==========================================
reduce in REPLAY-SCHEME : initQState |= [H(1) ; CX(1,2) ; H(3) ; CX(3,4) ; CX(
    0,1) ; H(0) ; if P(1,|0>) with prob >= 1/2 then skip else X(2) fi ; if P(0,
    |0>) with prob >= 1/2 then skip else Z(2) fi ; CX(2,3) ; H(2) ; if P(3,|0>)
    with prob >= 1/2 then skip else X(4) fi ; if P(2,|0>) with prob >= 1/2 then
    skip else Z(4) fi]P(4,qubitAt(initQState, 0)) .
result JudgmentSet: emptyJS
Bye.
